binding.cc

```diff
@@ -10,6 +10,24 @@
 #include "deps/transmission/libtransmission/utils.h"
 #include "deps/transmission/libtransmission/variant.h"
 
+#define MEM_K 1024
+#define MEM_K_STR "KiB"
+#define MEM_M_STR "MiB"
+#define MEM_G_STR "GiB"
+#define MEM_T_STR "TiB"
+
+#define DISK_K 1000
+#define DISK_K_STR "kB"
+#define DISK_M_STR "MB"
+#define DISK_G_STR "GB"
+#define DISK_T_STR "TB"
+
+#define SPEED_K 1000
```

Comment:
Memory is always a power of 2. Ex: 1 KiB of RAM is really 1024 B of RAM, not 1000 B.
Meanwhile IIUC storage is traditionally expressed as power of 10. so 1 kB is 1000 B.

To add confusion, OS and softwares often use "kB" (1000 B) instead of "KiB" (1024 B)

I just used Transmission's default units here, but it's debatable. See https://github.com/transmission/transmission/issues/198